frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -1,22 +1,64 @@
 <template lang='pug'>
-.c-send
+.c-send.inputgroup(:class='{"is-editing": isEditing}')
+  .c-replying(v-if='replyingMessage')
+    i18n Replying to {{replyingTo}}:&nbsp
+    | "{{ replyingMessage }}"
+    button.c-clear.is-icon-small(
+      :aria-label='L("Stop replying")'
+      @click='$emit("stopReplying")'
+    )
+      i.icon-times
+
   textarea.textarea.c-send-textarea(
     ref='textarea'
     :disabled='loading'
-    :placeholder='customSendPlaceholder'
+    :placeholder='L("Write your message...")'
```

Comment:
<img width="650" alt="Screen Shot 2020-12-02 at 1 31 37 PM" src="https://user-images.githubusercontent.com/138706/100933840-c5a1e580-34a2-11eb-88c1-85aa7862d3e8.png">

In Firefox Developer Edition, this scrollbar appears. Is there a way to make it go away?